Tasks/DeleteFilesV1/deletefiles.ts

```diff
@@ -70,6 +79,18 @@ tl.setResourcePath(path.join(__dirname, 'task.json'));
     // apply the match patterns
     let matches: string[] = tl.match(foundPaths, patterns, null, matchOptions);
 
+    
+
+    // Remove parent directories of excluded files from matches 
+    const removeExcludedDirectories = (matches: string[], paths: string[]): string[] => {
```

Comment:
Why should we exclude all content of parent directories from exlcuded paths? Does it support patterns like described [here](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/delete-files?view=azure-devops#delete-all-but-one-subdirectory)?

Reply:
Ok, as per discussion - it's required for avoiding of deletion of parent folders - of excluded files, so they still won't be deleted